src/modules/Extension/html_admin/mod_extension_languages.html.twig

```diff
@@ -1,73 +1,158 @@
+                                            href="{{ 'api/admin/extension/toggle_language'|link({ 'locale_id': lang.locale }) }}"
+                                            data-api-reload="1">
+                                            <svg class="icon">
+                                                <use xlink:href="#check" />
+                                            </svg>
+                                        </a>
+                                    </td>
+                                </tr>
+                            </tbody>
+                            {% else %}
+                                <tbody>
+                                <tr>
+                                    <td class="text-muted" colspan="2">{{ 'The list is empty'|trans }}
+                                    </td>
+                                </tr>
+                                </tbody>
+                            {% endfor %}
+                        </table>
+                    </div>
                 </div>
 
                 <div class="tab-pane fade" id="tab-new" role="tabpanel">
                     {% include "partial_extensions.html.twig" with { 'type': 'translation', 'header': 'List of translations on extensions site'|trans } only %}
                     <div class="card-body">
                         <h3 class="card-title">{{ 'FOSSBilling in your language'|trans }}</h3>
-                        <p class="card-subtitle">{{ 'Although FOSSBilling displays in U.S. English by default, the software has the built-in capability to be used in any language. Follow instructions bellow to install new language.'|trans }}</p>
+                        <p class="card-subtitle">{{ 'Although FOSSBilling displays in U.S. English by default, it has the capability to be used in any language. Follow instructions bellow to install new language.'|trans }}</p>
                         <div>
                             <ul>
-                                <li>{{ 'Check if your language translation file is available at'|trans }} <a href="https://translate.fossbilling.org" target="_blank">{{ 'FOSSBilling translations repository'|trans }}</a></li>
-                                <li>{{ 'Download language files and place them to'|trans }} <code>{{ constant('PATH_LANGS') }}</code></li>
-                                <li>{{ 'Language will be automatically available and can be changed in client area'|trans }}</li>
+                                <li>{{ 'Check if your language translation files are available at the '|trans }} <a href="https://github.com/FOSSBilling/locale" target="_blank">{{ 'FOSSBilling translations repository'|trans }}</a></li>
+                                <li>{{ 'Download the language files and place them under'|trans }} <code>{{ constant('PATH_LANGS') }}</code></li>
+                                <li>{{ 'The language will automatically be detected and made available for use.'|trans }}</li>
```

Comment:
it will *be* available for use.

Reply:
What even is your point here?
"The language will automatically be detected and made available for use." is perfectly understandable and is grammatically correct.

Reply:
> What even is your point here? "The language will automatically be detected and made available for use." is perfectly understandable and is grammatically correct.

Yep that reads well to me, it's clear and concise.